dotenv.sh

```diff
@@ -1,12 +1,25 @@
 #!/bin/sh
 set -e
 
+log_verbose() {
+	if [ "$VERBOSE" = 1 ]; then
+		echo "[dotenv.sh] $1" >&2
+	fi
+}
+
+is_set() {
+	eval val=\""\$$1"\"
```

Comment:
could be `[ -n "$(eval "echo \$$1")" ]` concisely.

Reply:
Combining line 11 and 12 sounds good